index.js

```diff
@@ -78,9 +80,16 @@ var run = function(options) {
         // JSON
         else {
           // some reports can be slimmed down for direct rendering
-          if (options.slim && report.slim) delete data.data;
-
-          writeReport(name, JSON.stringify(data, null, 2), ".json", done);
+          if (options.slim && report.slim) {
+            delete data.data;
+            writeReport(name, JSON.stringify(data, null, 2), ".json", done);
+          } else if (options["write-to-database"]) {
+            writeResultsToDatabase(data, { realtime: report.realtime }).then(() => {
+              writeReport(name, JSON.stringify(data, null, 2), ".json", done);
+            }).catch(err => done(err))
+          } else {
+            writeReport(name, JSON.stringify(data, null, 2), ".json", done);
```

Comment:
Nit: We now use `JSON.stringify(data, null, 2)` 3 times instead of 1, so I think we should do that once above this block and store it in a var, to avoid repeating the stringification config.